client/components/ChallengesList.js

```diff
@@ -1,24 +1,44 @@
 import React, { useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
-import { fetchChallenges } from '../store';
+import { fetchChallenges, fetchFilterChallenges, fetchSrchFltrChal } from '../store';
```

Comment:
let's use complete words when naming variables/functions.
```suggestion
import { fetchChallenges, fetchFilterChallenges, fetchChallengesWithFilterAndSearch } from '../store';
```